config/OPM-Reconfig.cfg

```diff
@@ -0,0 +1,499 @@
+
+ 		centralBody = Jool
+
+ 		spawnRate = 0.10
+
+ 		orbitSize
+		{
+			dist = Uniform
+			min = Ratio(Tylo.sma, 2)
+			max = Ratio(Pol.sma, 1)
+		}
+
+ 		eccentricity
+		{
+			dist = Uniform
+			avg = 0.2
+			stddev = 0.5
+		}
+
+ 		inclination
+		{
+			dist = Uniform
+			min = 0
+			max = 180
+		}
+
+ 		ascNode
+		{
+			dist = Uniform
+			min = 0
+			max = 360
+		}
+
+ 		periapsis
+		{
+			dist = Uniform
+			min = 0
+			max = 360
+		}
+
+ 		orbitPhase
+		{
+			dist = Uniform
+			min = 0
+			max = 360
+		}
+
+ 		asteroidTypes
+		{
+			key = 0.75 PotatoRoid
+			key = 0.20 CaAsteroidCarbon
+			key = 0.05 CaAsteroidMetal
+		}
+	}
+
+ 	ASTEROIDGROUP
+	{
+		name = sarnusOuter
+		title = #autoLOC_CustomAsteroids_OPM_GroupOuterObj
+
+ 		centralBody = Sarnus
+
+ 		spawnRate = 0.10
+
+ 		orbitSize
+		{
+			dist = Uniform
+			min = Ratio(Slate.sma, 2)
+			max = Ratio(Tekto.sma, 2)
+		}
+
+ 		eccentricity
+		{
+			dist = Uniform
+			avg = 0.2
+			stddev = 0.5
+		}
+
+ 		inclination
+		{
+			dist = Uniform
+			min = 0
+			max = 180
+		}
+
+ 		ascNode
+		{
+			dist = Uniform
+			min = 0
+			max = 360
+		}
+
+ 		periapsis
+		{
+			dist = Uniform
+			min = 0
+			max = 360
+		}
+
+ 		orbitPhase
+		{
+			dist = Uniform
+			min = 0
+			max = 360
+		}
+
+ 		asteroidTypes
+		{
+			key = 0.75 PotatoRoid
+			key = 0.20 CaAsteroidCarbon
+			key = 0.05 CaAsteroidMetal
+		}
+	}
+
+ 	ASTEROIDGROUP
+	{
+		name = urlumOuter
+		title = #autoLOC_CustomAsteroids_OPM_GroupOuterObj
+
+ 		centralBody = Urlum
+
+ 		spawnRate = 0.10
+
+ 		orbitSize
+		{
+			dist = Uniform
+			min = Ratio(Priax.sma, 2)
+			max = Ratio(Wal.sma, 2)
+		}
+
+ 		eccentricity
+		{
+			dist = Uniform
+			avg = 0.2
+			stddev = 0.5
+		}
+
+ 		inclination
+		{
+			dist = Uniform
+			min = 0
+			max = 180
+		}
+
+ 		ascNode
+		{
+			dist = Uniform
+			min = 0
+			max = 360
+		}
+
+ 		periapsis
+		{
+			dist = Uniform
+			min = 0
+			max = 360
+		}
+
+ 		orbitPhase
+		{
+			dist = Uniform
+			min = 0
+			max = 360
+		}
+
+ 		asteroidTypes
+		{
+			key = 0.10 CaAsteroidCarbon
+			key = 0.90 CaAsteroidIcy
+		}
+	}
+
+ 	ASTEROIDGROUP
+	{
+		name = neidonOuter
+		title = #autoLOC_CustomAsteroids_OPM_GroupOuterObj
+
+ 		centralBody = Neidon
+
+ 		spawnRate = 0.15
+
+ 		orbitSize
+		{
+			dist = Uniform
+			min = Ratio(Thatmo.sma, 2)
+			max = Ratio(Nissee.sma, 1)
+		}
+
+ 		eccentricity
+		{
+			dist = Uniform
+			avg = 0.2
+			stddev = 0.5
+		}
+
+ 		inclination
+		{
+			dist = Uniform
+			min = 0
+			max = 180
+		}
+
+ 		ascNode
+		{
+			dist = Uniform
+			min = 0
+			max = 360
+		}
+
+ 		periapsis
+		{
+			dist = Uniform
+			min = 0
+			max = 360
+		}
+
+ 		orbitPhase
+		{
+			dist = Uniform
+			min = 0
+			max = 360
+		}
+		
+ 		asteroidTypes
+		{
+			key = 0.10 CaAsteroidCarbon
+			key = 0.90 CaAsteroidIcy
+		}
+	}
+}
+
+Localization
+{
+	en-us
+	{
+    #autoLOC_CustomAsteroids_OPM_GroupInnerObj = Gas Giant Moonlet
+    #autoLOC_CustomAsteroids_OPM_GroupOuterObj = Captured Asteroid
```

Comment:
Consider making these abbreviations (e.g., "Captured Ast."), both for consistency with the stock style and to keep the full asteroid name short.